.github/workflows/push_deploy.yaml

```diff
@@ -47,5 +47,12 @@ jobs:
       - name: cleanup container
         if: always()
         run: |
+          image="quay.io/centos/centos:stream9"
           podman kill anabot
-          podman image rm quay.io/centos/centos:stream9
```

Comment:
Couldn't this be done simply by `podman image rm --force ...` ?

Reply:
Yes, it should work the same, I think. I may try to resort to the force deletion, but I wonder why the image still doesn't get deleted by the current solution, despite it not being used by any container (as you can see in the workflow script output). @velezd, you know most likely much more about the GitHub workflows - is it possible that this is a 'feature' for some reason?

Reply:
@jstodola so I tried to just enforce the image removal as you suggested and it works fine (as you can see in the checks below). However, just FYI, I think that even if we didn't do any cleanup at all, it shouldn't actually cause any mess, because the workflows [use a fresh VM](https://docs.github.com/en/actions/about-github-actions/understanding-github-actions#runners) every time, which makes sense after all:

> Each workflow run executes in a fresh, newly-provisioned virtual machine.